index.js

```diff
@@ -114,105 +117,109 @@ async function run() {
+  const websiteSha = await getHeadSha( "digital-canada-ca-website", "main" );
+  branchName = `content-release-${ new Date().getTime() }`;
 
-  let refs = await octokit.git.createRef({
-    owner: 'cds-snc',
-    repo: 'digital-canada-ca-website',
-    ref: `refs/heads/${branchName}`,
+  let refs = await octokit.git.createRef( {
+    owner: REPO_CONFIG.owner,
+    repo: REPO_CONFIG.repo,
+    ref: `refs/heads/${ branchName }`,
     sha: websiteSha
-  });
-
+  } );
 
   // Create / Update file commits
   // Blog posts
-  await createAndUpdateFiles(blogPostsEnNew, existingContentEN.data.tree, "en", "blog/posts/", branchName);
-  await createAndUpdateFiles(blogPostsFrNew, existingContentFR.data.tree, "fr", "blog/posts/", branchName);
-  await createAndUpdateFiles(gcArticlesBlogsEn, existingContentEN.data.tree, "en", "blog/posts/", branchName);
-  await createAndUpdateFiles(gcArticlesBlogsFr, existingContentFR.data.tree, "fr", "blog/posts/", branchName);
+  await createAndUpdateFiles( blogPostsEnNew, existingContentEN.data.tree, "en", "blog/posts/", branchName );
+  await createAndUpdateFiles( blogPostsFrNew, existingContentFR.data.tree, "fr", "blog/posts/", branchName );
+  await createAndUpdateFiles( gcArticlesBlogsEn, existingContentEN.data.tree, "en", "blog/posts/", branchName );
+  await createAndUpdateFiles( gcArticlesBlogsFr, existingContentFR.data.tree, "fr", "blog/posts/", branchName );
   // Job Postings
-  await createAndUpdateFiles(jobPostsEnNew, existingContentEN.data.tree, "en", "jobs/positions/", branchName);
-  await createAndUpdateFiles(jobPostsFrNew, existingContentFR.data.tree, "fr", "jobs/positions/", branchName);
-  await createAndUpdateFiles(gcArticlesJobPostsEn, existingContentEN.data.tree, "en", "jobs/positions/", branchName)
-  await createAndUpdateFiles(gcArticlesJobPostsFr, existingContentFR.data.tree, "fr", "jobs/positions/", branchName)
-
+  await createAndUpdateFiles( jobPostsEnNew, existingContentEN.data.tree, "en", "jobs/positions/", branchName );
+  await createAndUpdateFiles( jobPostsFrNew, existingContentFR.data.tree, "fr", "jobs/positions/", branchName );
+  await createAndUpdateFiles( gcArticlesJobPostsEn, existingContentEN.data.tree, "en", "jobs/positions/", branchName )
+  await createAndUpdateFiles( gcArticlesJobPostsFr, existingContentFR.data.tree, "fr", "jobs/positions/", branchName )
+  // Pages
+  await createAndUpdateFiles( gcArticlesPagesEn, existingContentEN.data.tree, "en", "pages/", branchName );
+  await createAndUpdateFiles( gcArticlesPagesFr, existingContentFR.data.tree, "fr", "pages/", branchName );
 
   // if there is content - compare shas of most recent commit on the branch and main
-  let branchcommit = await octokit.request('GET /repos/{owner}/{repo}/commits/{sha}', {
-    owner: 'cds-snc',
-    repo: 'digital-canada-ca-website',
-    sha: branchName
+  let branchcommit = await octokit.repos.getBranch({
+    owner: REPO_CONFIG.owner,
+    repo: REPO_CONFIG.repo,
+    branch: branchName
```

Comment:
Our of curiosity, would it make sense to use this new method for getting  the `branchCommit` to get the `mainCommit` further down?